.vib/fluxcd-helm-controller/goss/fluxcd-helm-controller.yaml

```diff
@@ -0,0 +1,6 @@
+command:
+  check-custom-daemon-help:
+    exec: helm-controller --help
+    exit-status: 2
```

Comment:
This might become flaky, I wonder if we just should remove this test.

Reply:
The rule of thumb here should be to check the command executes correctly (`exit-status: 0`), preferring that over specific output checks (which also may become flaky if the app changes).

This case is strange, I'd say that we should:

1. Remove the `exit-status: 2` check. We don't really care what the actual number is (other than 0)
2. Leave the `stderr` check. If this is eventually modified, we will update it.